tokio/CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+# 1.41.0 (Oct 19th, 2024)
+
+### Added
+
+- metrics: stabilize `global_queue_depth` ([#6854], [#6918])
+- net: add conversions for unix `SocketAddr` ([#6868])
+- sync: add `watch::Sender::sender_count` ([#6836])
+- sync: add `mpsc::Receiver::blocking_recv_many` ([#6867])
+- task: stabilize `Id` apis ([#6793], [#6891])
+
+### Added (unstable)
+
+- metrics: add H2 Histogram option to improve histogram granularity ([#6897])
+- runtime: add `LocalRuntime` ([#6808])
+
+### Changed
```

Comment:
We merged https://github.com/tokio-rs/tokio/pull/6924 too.

Sorry for so many edits :) let's not make any more.